sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -434,7 +434,6 @@ abstract class ScrollableListPainter extends Painter {
   double _scrollOffset = 0.0;
   void set scrollOffset (double value) {
     assert(value != null);
-    assert(value >= 0.0 && value <= 1.0);
```

Comment:
If you use the 'flutter' tool, it defaults to checked. sky_tool is deprecated now.